src/components/CharClass.js

```diff
@@ -0,0 +1,22 @@
+import '../css/CharClass.css';
+import { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import Bookmark from './Bookmark';
+import useIcon from '../hooks/useIcon';
+
+const CharClass = ({ charClass }) => {
+  const { index, name } = charClass;
+  const { icon } = useIcon(index);
```

Comment:
This really is a silly solution... I'm saying this as the person who wrote this silly hooks, just FYI 😂